e2e/__tests__/init.test.js

```diff
@@ -84,3 +84,17 @@ test('init --template file:/tmp/custom/template', () => {
 
   expect(stdout).toContain('Run instructions');
 });
+
+test('init --verbose', () => {
+  const {stdout} = run(DIR, [
+    'init',
+    '--template',
+    'react-native-new-template',
+    'TestInit',
+    '--verbose',
```

Comment:
I don't think we need a test with verbosity on, not a full one at least. Maybe you could use `runUntil` to pause execution after certain log output is there or create a unit test instead?

Reply:
The main problem I have is that running multiple `run`s in one test gets _very_ slow because there's no parallelization there.

Reply:
I don't think this test is necessary.

Reply:
kill it